### ISSUE TITLE
install fails if git protocol blocked

### ISSUE THREAD

**Issue by twellspring** [Strider-CD/strider#909]
Install of Srider-CD  via `npm install -g strider`  fails if the git protocol is blocked by firewalls.

```npm ERR! git clone --template=/root/.npm/_git-remotes/_templates --mirror git://github.com/janruehling/everypaas.git /root/.npm/_git-remotes/git-github-com-janruehling-ever
ypaas-git-fixHerokuLookup-abeea96f0e03c9e3d3e3e751627484e9: github.com[0: 192.30.252.129]: errno=Connection timed out```

Cause of this is the explicit git dependency in package.json
``` "everypaas": "git://github.com/janruehling/everypaas.git#fixHerokuLookup",```

Could this be replaced with the the npm everypass module (from niallo)?

**Comment by knownasilya** [comment]
The PR hasn't been merged yet. Feel free to make a note in the PR.

**Comment by knownasilya** [comment]
Temporary workaround might be to remove the git protocol and just make it https, it should work as well. Or change it to `janruehling/everypaas.git#fixHerokuLookup`.

**Comment by knownasilya** [comment]
Try installing from master, `npm install -g Strider-CD/strider`.

**Comment by twellspring** [comment]
I worked around it temporarily by disabling my development firewall.  But thanks for the fix above.  I will try that when I put it into prod.

**Comment by niallo** [comment]
@knownasilya I published `everypaas` 0.0.8 to npm. https://www.npmjs.com/package/everypaas
